susemanager-utils/susemanager-sls/salt/virt/create-vm.sls

```diff
@@ -121,3 +121,32 @@ domain_define:
     - require:
       - virt: domain_first_boot_define
 {%- endif %}
+
+{%- if pillar.get("cluster_definitions") %}
+'virsh dumpxml --inactive {{ pillar['name'] }} >{{ pillar['cluster_definitions'] }}/{{ pillar['name'] }}.xml':
```

Comment:
Hmmm, I would suggest to not use the command here directly as the state id. Better do something like:

```
dump-virsh-inactive-xml-to-pillar-whatever:
  cmd.run:
    - name: "virsh dumpxml --inactive {{ pillar['name'] }} >{{ pillar['cluster_definitions'] }}/{{ pillar['name'] }}.xml"
    - require:
      - virt: {{ "domain_define" if cdrom_boot or ks_boot else "domain_first_boot_define" }}
```

IMHO this way it looks more readable and also state id gives you some hints on what the state is actually doing.

Reply:
ACK, will change